internal/pkg/mount/v2/pseudo.go

```diff
@@ -43,5 +42,12 @@ func PseudoSubMountPoints() Points {
 		)
 	}
 
+	if _, err := os.Stat("/sys/fs/selinux"); err == nil {
+		// mount selinuxfs if it exists
```

Comment:
In which cases it's not present? When using AppArmor as the major LSM?

Reply:
yes, when lsm  is apparmor